SignalShareExtension/utils/ShareAppExtensionContext.m

```diff
@@ -145,7 +145,6 @@ - (void)setStatusBarHidden:(BOOL)isHidden animated:(BOOL)isAnimated
 
 - (CGFloat)statusBarHeight
 {
-    OWSFail(@"%@ in %s unexpected for share extension", self.logTag, __PRETTY_FUNCTION__);
```

Comment:
What calls this now?

Reply:
SAE uses statusBarHeight via OWSNavigationController via OWSNavbar